### ISSUE TITLE
Is there a node for Kubernetes itself?

### ISSUE THREAD

**Issue by flyisland** [mingrammer/diagrams#979]
I've checked the page https://diagrams.mingrammer.com/docs/nodes/k8s but found no one for Kubernetes itself

**Comment by filipeaaoliveira** [comment]
Hello @flyisland 

Are you talking about this one https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png ?

Definitely should be added.

I think that under under K8s / Ecossystem. 
What do you think @gabriel-tessier @tvqphuoc01 ?

**Comment by flyisland** [comment]
Yes,  https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png is what I'm talking about.

**Comment by filipeaaoliveira** [comment]
I've assigned it to @gabriel-tessier
